src/main/java/org/sonarsource/sonarlint/ls/backend/BackendService.java

```diff
@@ -187,6 +187,12 @@ public CompletableFuture<GetBindingSuggestionsResponse> getBindingSuggestion(Get
     return initializedBackend().getBindingService().getBindingSuggestions(params);
   }
 
+  public CompletableFuture<org.sonarsource.sonarlint.core.clientapi.backend.issue.CheckStatusChangePermittedResponse>
+  checkStatusChangePermitted(String connectionId, String issueKey) {
```

Comment:
For consistency with existing methods, should we pass in `CheckStatusChangePermittedParams params` rather than `String connectionId, String issueKey`?